runtime/tools/spec-builder/src/modules/aura.js

```diff
@@ -0,0 +1,53 @@
+const { u8aToHex } = require('@polkadot/util');
+const { Keyring } = require('@polkadot/api');
+const { __retain, __getUint8Array, __allocArray, __release, UInt8Array_ID, getAccountIdBytes, getAuraKey, UInt, __reset } = require('../wasm-loader');
+
+
+class Aura {
+
+    /**
+     * Converts key&value pair to scale-encoded type
+     * @param authorities list of authorities
+     */
+    static toRaw(authorities){
+        let rawAuthorities = [];
```

Comment:
I think that it would be best to check that the authorities that are passed to this `toRaw` function are:
1. array
2. not empty